src/dcc.c

```diff
@@ -454,30 +454,24 @@ static void free_dcc_bot_(int n, void *x)
 static void out_dcc_bot(int idx, char *buf, void *x)
 {
   size_t len = strlen(buf);
-  /* We don't really use x here, so "use it" for the compiler */
-  (void)x;
```

Comment:
Can you confirm -Wunused-parameter won't complain about removing this?

Reply:
switching `-Wunused-parameter` on, reports 100s of such cases within eggdrops code base, so i dont see why this should be a problem with this specific function.